packages/block-library/src/file/edit.js

```diff
@@ -101,11 +116,14 @@ function FileEdit( { attributes, setAttributes, noticeUI, noticeOperations } ) {
 	function onSelectFile( newMedia ) {
 		if ( newMedia && newMedia.url ) {
 			setHasError( false );
+			const isPdf = newMedia.url.endsWith( '.pdf' );
 			setAttributes( {
 				href: newMedia.url,
 				fileName: newMedia.title,
 				textLinkHref: newMedia.url,
 				id: newMedia.id,
+				displayPreview: isPdf ? true : undefined,
+				previewHeight: isPdf ? 800 : undefined,
```

Comment:
Since we changed the default..
```suggestion
				previewHeight: isPdf ? 600 : undefined,
```

Reply:
Ooops! I knew I'd hardcoded it somewhere else, but couldn't find it earlier. 🙃